### ISSUE TITLE
Add ability to send private replies.

### ISSUE THREAD

**Issue by MarcDavid99** [xatkit-bot-platform/labs-ut-platform#18]
https://developers.facebook.com/docs/messenger-platform/discovery/private-replies/
Facebook for Developers

**Comment by Raud0** [comment]
The feature is blocked by #16 It requires `recipient.post_id` and `recipient.comment_id` to be added to context. Since this functionality wasn't directly requested by the developers of Xatkit, I'm making this issue low priority.

**Comment by Raud0** [comment]
Closing culled issue.
